src/components/CommandMenu/CommandMenuPanel.tsx

```diff
@@ -0,0 +1,87 @@
+import SwipeableDrawer from '@mui/material/SwipeableDrawer'
+import _ from 'lodash'
+import { useRef } from 'react'
+import { useDispatch, useSelector } from 'react-redux'
+import { css } from '../../../styled-system/css'
+import { token } from '../../../styled-system/tokens'
+import { toggleDropdownActionCreator } from '../../actions/toggleDropdown'
+import { isTouch } from '../../browser'
+import isTutorial from '../../selectors/isTutorial'
+import durations from '../../util/durations'
+import CloseIcon from '../icons/CloseIcon'
+import PanelCommandGrid from './PanelCommandGrid'
+
+/**
+ * A panel that displays the command menu.
+ */
+const CommandMenuPanel = () => {
+  const dispatch = useDispatch()
+  const showCommandMenu = useSelector(state => state.showCommandMenu)
+  const isTutorialOn = useSelector(isTutorial)
+  const containerRef = useRef<HTMLInputElement>(null)
+
+  if (isTouch && !isTutorialOn) {
+    return (
+      <SwipeableDrawer
+        data-testid='command-menu-panel'
+        // Disable swipe to open - this removes the swipe-up-to-open functionality
+        disableSwipeToOpen={true}
+        ref={containerRef}
```

Comment:
Looks like `containerRef` is no longer used and can be removed. Leftover from `Sidebar` code.